release/nightly_tests/stress_tests/test_state_api_scale.py

```diff
@@ -78,7 +78,7 @@ def test_many_tasks(num_tasks: int):
     #  where num_total_cpus obtained from cluster_compute in `release_tests.yaml`
     # 1 / (10k/45 * 7) ~= 0.03, taking a smaller value to make sure all tasks could be
     # running at the same time.
-    @ray.remote(num_cpus=0.02)
+    @ray.remote(num_cpus=0.03)
```

Comment:
we should actually make it 0.002? 

It was 10k max, so now, 1 / (100k/45 *7) ~= 0.003?